data/json/recipes/other/tool.json

```diff
@@ -134,9 +134,8 @@
     "type": "recipe",
     "category": "CC_OTHER",
     "subcategory": "CSC_OTHER_TOOLS",
-    "skill_used": "fabrication",
-    "difficulty": 4,
-    "skills_required": [ "fabrication", 4 ],
+    "skill_used": "electronics",
+    "difficulty": 3,
```

Comment:
Please tell what do you want to implement.
Fabrication 4 _and_ electronics 3? Or just electronics 3?